modules/dnn/test/test_ie_models.cpp

```diff
@@ -276,6 +280,21 @@ void runCV(Backend backendId, Target targetId, const std::string& xmlPath, const
     }
 }
 
+inline static std::string getOpenVINOModel(const std::string &modelName, bool isFP16)
+{
+    const std::map<std::string, OpenVINOModelTestCaseInfo>& models = getOpenVINOTestModels();
+    const auto it = models.find(modelName);
+    if (it != models.end())
+    {
+        OpenVINOModelTestCaseInfo modelInfo = it->second;
+        if (isFP16 && modelInfo.modelPathFP16)
+            return std::string(modelInfo.modelPathFP16);
+        else if (!isFP16 && modelInfo.modelPathFP32)
+            return std::string(modelInfo.modelPathFP32);
+    }
+    return std::string();
```

Comment:
IMHO, error with description or skip action should be raised.
Test code should not allow to hide mistakes.

Reply:
I wanted this routine to be separate from testing framework to allow calling it from some static initializer or test fixture constructors. Problems should be caught later by the test body in `findDataFile` and `string::empty()` checks.